### PR TITLE
Feature/amrit s/1778816899 home contact backend sync and population

### PR DESCRIPTION
**Pull request by Amrit-S**
### Administrative Info
Monday Board ID: 1181608900
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Synced Home page to backend (includes News & Events + Branches & Chapters)
- Populated Home & Contact page with relevant site information 
- Slightly restructured Home page (added in additional sections), and updated Interactive Map (zooming enabled only for mobile, map is bigger, and more centered)
- Created responsive Donate page (PayPal Button synced to Sakyadhita's own account)
- Created Responsive 404 Page 
- Updated Footer (added icons, subpages, and animations)
- Added animations on Home page, Footer, Newsletters, EPubs, and Buddhist Culture/About Us pages
- Updated all image headers to Dhanush's ResourceHeader component for consistency 
- Backend: Made email attribute for Branches & Chapters optional due to client demand 

### Testing
How did you confirm your changes worked? 
- Tested on various screen sizes 

### Confirmation of Change 

https://user-images.githubusercontent.com/42155577/142183349-4d352eb2-6959-40c9-8ba9-aacee83b0bd0.mp4

Screen Shot 2021-11-17 at 2 24 00 AM
Screen Shot 2021-11-17 at 2 23 21 AM
Screen Shot 2021-11-17 at 2 26 04 AM
Screen Shot 2021-11-17 at 2 23 43 AM

### CODE REVIEW

**Review by amiteshksharma — CHANGES_REQUESTED** [review]
Nice PR. Gotta say the underline color change on hover for the footer elements is spicy. Couple of things though: 

1) When you click on a location marker and then scroll, the little pop-up stays on the screen and you can have something like this show up. Probably not an ideal feature but if you want it then it is what it is. 

image

2) I assume you know but just want to write it down, some image URLs do not work on the donate page. 

image

3) Not sure if this change was part of your PR or not, but for Buddhist Culture the side navbar starts inside the header photo as seen below. 

image

4) For the Join us, Volunteer, and Donate page, there is a circle in the bottom right of the image header but I can't click it or anything, so is there a reason for this? or is it just to indicate to scroll down

image

5) Also, for the header on "Join us", "Donate & Support", and "Volunteer", we have the filler text with random words. I assume you are waiting on them to add the proper text but I'll write it down here just so we remember. 

image

**Comment by aaron-kirk** [comment]
the slider thing was still broken for me so I changed it such that I think it should be fixed? works for me at least...

sorry for my hacky code 😅😅

File: seeds/models/newsAndEvents.js [hunk 1]
```diff
@@ -4,52 +4,73 @@ module.exports = async () => {
...
+            },
+            { logging: false }
+        ),
+        newsAndEvents.create(
+            {
+                title: "New Newsletter",
+                description: "Check out the latest Sakyadhita Newsletter!",
+                imageLink: "https://www.dropbox.com/s/dtuggjn3ouuaul5/Sakyadhita%20Newsletters.jpeg?raw=1",
+                redirectLink:
+                    "https://sakyadhita.org/docs/resources/newsletters/2021%20Sakyadhita%20Newsletter%20Vol.%2029.pdf",
```
Comment (amiteshksharma) [inline seeds/models/newsAndEvents.js:59]: Is someone else adding newsletters to the dropbox?

**Review by aaron-kirk — APPROVED** [review]
LPTM -- so much stuff, but I couldn't really find any issues. This is why you're the #1 PM.

**Review by amiteshksharma — APPROVED** [review]
What a great PR. A couple of small bugs at the start, but what a great comeback here. You really pulled not only your weight, but the weight of the whole damn team. I am thoroughly impressed by the work done in this PR. Can't wait to merge this into main and get one step closer to finishing this project! Great job Amrit, where would this team be without your experience, expertise, and guidance.

**Pull request merged**